OpenSim/Actuators/Test/testMuscles.cpp

```diff
@@ -1108,10 +929,95 @@ void testDelp1990Muscle()
         x0, 
         act0, 
         &motion, 
-        &control, 
-        IntegrationAccuracy,
-        SimulationTest,
-        SimulationTestTolerance,
+        &control,
         false);
+}
+
+
+
+void testMuscleEquilibriumSolve(const Model& model, const Storage& statesStore)
+{
+    // Get the muscle to test
+    const Muscle& muscle = model.getMuscles()[0];
+
+    if (dynamic_cast<const Millard2012AccelerationMuscle*>(&muscle)) {
+        // Millard2012AccelerationMuscle is not an Equilibrium-based Muscle
+        return;
+    }
+
+    // Load input data as StatesTrajectory used to perform the Analysis
+    auto statesTraj = StatesTrajectory::createFromStatesStorage(
+        model, statesStore, true, false);
+    size_t nstates = statesTraj.getSize();
 
+    // muscle active, passive, total muscle and tendon force quantities
+    double af, pf, mf, tf, nfl, fv;
+    af = pf = mf = tf = nfl = fv = SimTK::NaN;
+
+    // Tolerance for muscle equilibrium solution 
+    const double equilTol = muscle.getMaxIsometricForce()*SimTK::SqrtEps;
+
+    // The maximum acceptable change in force between two contiguous states
+    const double maxDelta = muscle.getMaxIsometricForce() / 2;
```

Comment:
The same line appears in two places. The one in `testAnalyzeTool` was addressed and this one was overlooked. Thank you for seeing that!